docrepr/webview.py

```diff
@@ -0,0 +1,55 @@
+# -*- coding: utf-8 -*
+
+"""
+View object documentation HTML in a separate GUI window.
+
+Requires PyQt4 or PySide. Original code by
+anatoly techtonik <techtonik@gmail.com> is placed in
+public domain.
```

Comment:
But this is not a script, nor something that can be used independently of the project. It's just an additional module. So please remove your name and the public domain notice out of it. The `AUTHORS` file already mention those who have worked in the project.

This has been the work of several people, who helped to fix bugs and improve things in the code inherited from Spyder. That's why in `AUTHORS` appears the whole Spyder Development Team as author.